src/Parsec/Shaiya/Wld/PatrolCoordinate.cs

```diff
@@ -1,5 +1,5 @@
-﻿using System.Numerics;
 using Parsec.Attributes;
+using Parsec.Shaiya.Common;
```

Comment:
Why is this needed here? I think you meant to add it somewhere else

Reply:
Nvm, it's a namespace change from System.Numerics.Vector3 -> Parsec.Shaiya.Common.Vector3